hnn_core/hnn_io.py

```diff
@@ -15,6 +15,244 @@
 from .cell_response import CellResponse
 from .externals.mne import fill_doc
 
+def network_to_markdown(net, fname=None):
+    """converts network parameters to a markdown table."""
+    from datetime import datetime
+    
+    md = []
```

Comment:
```suggestion
    md = list()
```

hard to distinguish between `[]` and `{}` otherwise